src/OmniSharp.Roslyn.CSharp/Services/Buffer/UpdateBufferService.cs

```diff
@@ -20,7 +20,7 @@ public UpdateBufferService(OmniSharpWorkspace workspace)
 
         public async Task<object> Handle(UpdateBufferRequest request)
         {
-            await _workspace.BufferManager.UpdateBuffer(request);
+            await _workspace.BufferManager.UpdateBufferAsync(request);
```

Comment:
Really Async naming???? 😲 

(However since we leverage Roslyn it makes sense to follow similar patterns here 👍 )